samples/artifacts/EventCollection/ReceiveNatsinTextFormat.siddhi

```diff
@@ -0,0 +1,51 @@
+@App:name("ReceiveNatsinTextFormat")
+@App:description('Receive events via NATS transport in text format with default mapping and view the output on the console.')
+
+/*
+Purpose:
+    This application demonstrates how to configure siddhi applications to receive events to the SweetProductionStream via NATS transport in text format with some regular expression and log the events in LowProductionAlertStream to the output console.
+
+Prerequisites:
+    1)Install the NATS Streaming server from "https://nats-io.github.io/docs/nats_streaming/gettingstarted/install.html"
+    2)The Siddhi app runtime will need the following JARs to connect to the NATS streaming server. Download those JARs using the following links and add those JARs to "{SiddhiHome}/jars" directory.
+        a)Java NATS streaming :-"https://mvnrepository.com/artifact/io.nats/java-nats-streaming/2.1.2"
+        b)JNATS:-"https://mvnrepository.com/artifact/io.nats/jnats/2.3.0"
+    3)It also needs the Protobuf bundle. Download the Protobuf bundle using the following link and add the bundle to "s{SiddhiHome}/bundles" directory.
+        a)Protobuf:-"https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java/3.6.1"
+        
+Configurations:
+      For local deployments to testing this sample, need set up these environmental variables as follows.
+        a)STAN_CLUSTER_ID="test-cluster"
```

Comment:
You don't need to set these environmental variables if you directly setting `cluster.id` and `bootstrap.servers` inline. Either remove the creation of this environmental variable from the description or use these envs in the NATS source.

Reply:
9f63596947b84bc1942360a9da56f478c9c1389b This commit fixed the above correction